docs/reference/mapping.asciidoc

```diff
@@ -262,3 +270,5 @@ include::mapping/fields.asciidoc[]
 include::mapping/params.asciidoc[]
 
 include::mapping/dynamic-mapping.asciidoc[]
+
+include::mapping/runtime.asciidoc[]
```

Comment:
I wonder if runtime fields should appear last on the right. Maybe they should be close to field data types and metadata fields? Also, mapping parameters does not apply to runtime fields, does it mean that maybe we need to restructure things a bit? I am not extremely sure what the answer is but i was curious if you had thought about how the overall mappings docs may evolve with the introduction of runtime fields. Before them, there were only indexed fields.

Reply:
I moved _Runtime fields_ up in the navigation, and also restructured the main _Mappings_ page. I still think this section needs more work beyond the inclusion of runtime fields.